xbmc/settings/VideoSettings.cpp

```diff
@@ -86,3 +86,49 @@ bool CVideoSettings::operator!=(const CVideoSettings &right) const
   if (m_VideoStream != right.m_VideoStream) return true;
   return false;
 }
+
+int CVideoSettings::GetNextViewMode(int viewMode)
```

Comment:
I don't like those methods. Assume someone wants to insert another view mode in the middle. First the enum in iPlayer has to be changed, then this entire switch/case has to be reorganized.
An array that maps the enum to the string would fit better. Example: https://github.com/xbmc/xbmc/blob/master/xbmc/GUIInfoManager.cpp#L206